### DISCUSSION TITLE
Contributions on bug fixes

### DISCUSSION THREAD

**Discussion by benlazaro** [otros-systems/otroslogviewer#629] [General]
I think we should have some guidelines on how to tackle bug fixes.
This is what I have in mind and would like some feedback for improving it.

1. A bug/issue needs to be reproducible before trying to fix it. (Request more info from the issue author if necessary)
2. Pull requests need to be approved by at least 2 people before merging it to the master. (3 people would be best for democratic votes)
3. Code collaborations need to include unit tests (unless not applicable, handled case by case)

What do you think about these? What else can be added?

**Comment by svennissel** [comment]
1. Yes - if the discretion is very bad, we do not intensive work on a bug report. But have a short look on it is every time a good idea
2. Yes we must do it. But at the moment we only have @benlazaro and @nuessgens as active developer. I think I will to start a discussion here if i want to found a reviewer. At the beginning we must to try to get minimum two reviewer. At the moment I no permissions to change the rights of the merge request workflow. But a comment like "I have tested it everything is fine.." at the pull request or issues is also OK.
3. We must create more Test in feature. I want to have something like >80% code coverage on new code. This is the reason to have a look on static code analyse and code coverage support. The best way i something like SonarQube, but we don't have a central build server. I want to check this in Feature manual
